### ISSUE TITLE
Does clearml support offline import?

### ISSUE THREAD

**Issue by yaoshanliang** [allegroai/clearml#1133]
Our server can't connect to an external network, but there is still a need to use clearml to analyze training. So does clearml support to generate an offline file so that I can import the file to a device that can be connected to the internet and then import it to the clearml platform.

**Comment by ainoam** [comment]
Yes, it does @yaoshanliang. See https://clear.ml/docs/latest/docs/clearml_sdk/task_sdk#offline-mode.

**Comment by yaoshanliang** [comment]
@ainoam Thank you. I have generated the offline zip file. When I upload this file, it outputs 'INFO - Failed model upload' as the path on the external network is different from that on my computer. How can I disable the model upload or make a relative path in the zip file.

**Comment by ainoam** [comment]
@yaoshanliang the ClearML storage handler supports [path substition](https://clear.ml/docs/latest/docs/integrations/storage#path-substitution) for such use cases: Configuring `registered_prefix` to match your original path and `local_prefix` to the desired one, before `import_offline_session()` is called should upload to the desired location.
